src/workerd/api/r2-api.capnp

```diff
@@ -214,6 +216,9 @@ struct R2HeadResponse {
 
   checksums @8 :R2Checksums;
   # If set, the available checksums for this object
+
+  storageClass @9 :Text;
+  # The storage class of the object. Standard or Infrequent Access.
```

Comment:
nit: the comment could be expanded a bit more to explain what a "storage class" is